test-utils/fixtures/MovementSubmissionFailureFixtures.scala

```diff
@@ -69,4 +70,28 @@ trait MovementSubmissionFailureFixtures extends BaseFixtures {
     hasBeenFixed = false
   )
 
+  val dispatchWarehouseExciseInvalid: MovementSubmissionFailure = MovementSubmissionFailure(
+    errorType = DispatchWarehouseExciseInvalid.code,
+    errorReason = "The excise ID for the tax warehouse of dispatch is not valid",
+    errorLocation = Some("/IE815[1]/Body[1]/SubmittedDraftOfEADESAD[1]/EadEsadDraft[1]/LocalReferenceNumber[1]"),
```

Comment:
It's probably worth being consistent with the expected value, but looks like other values in this file don't do that, so may be worth setting this to None?